tests/bsim/bluetooth/host/gatt/general/src/gatt_client_test.c

```diff
@@ -54,7 +54,8 @@ static void disconnected(struct bt_conn *conn, uint8_t reason)
 
 	bt_conn_unref(g_conn);
 
-	g_conn = NULL;
+	__ASSERT_NO_MSG(g_conn == conn);
+
```

Comment:
Should this have been at line 39?

Setting `g_conn = NULL` after an unref seems like the right thing to do.

Reply:
yes, sorry

Reply:
Thank you for catching this!

Reply:
this is fixed now